source/config/configSpec.py

```diff
@@ -235,6 +235,8 @@
 	# 0: Off, 1: Speech, 2: Tones, 3: Both Speech and Tones
 	reportLineIndentation = integer(0, 3, default=0)
 	ignoreBlankLinesForRLI = boolean(default=False)
+	# Duration of indentation beeps, in milliseconds
+	indentToneDuration = integer(min=1, default=40)
```

Comment:
Also make sure to define a max.
```suggestion
	indentToneDuration = integer(min=1, max=200, default=60)
```

Reply:
I'm not against adding a maximum, but practically speaking, why do you think it is necessary? Why have you chosen 200ms?